db/osqlcomm.c

```diff
@@ -6436,6 +6436,7 @@ int osql_process_packet(struct ireq *iq, unsigned long long rqid, uuid_t uuid,
             return conv_rc_sql2blkop(iq, step, -1, ERR_NO_SUCH_TABLE, err,
                                      tablename, 0);
         }
+        iq->origdb = iq->usedb;
```

Comment:
Neither way is particularly clean.  I'm ok with either.  If origdb is confusing, let's set usedtableversion before calling into record.c.  Either way works.